<!DOCTYPE html><html lang="en"><head><meta charset="utf-8"><meta name="viewport" content="width=device-width, initial-scale=1.0"><meta name="generator" content="rustdoc"><meta name="description" content="Exceptions defined in Python’s `asyncio` module."><title>pyo3::exceptions::asyncio - Rust</title><script>if(window.location.protocol!=="file:")document.head.insertAdjacentHTML("beforeend","SourceSerif4-Regular-6b053e98.ttf.woff2,FiraSans-Regular-0fe48ade.woff2,FiraSans-Medium-e1aa3f0a.woff2,SourceCodePro-Regular-8badfe75.ttf.woff2,SourceCodePro-Semibold-aa29a496.ttf.woff2".split(",").map(f=>`<link rel="preload" as="font" type="font/woff2" crossorigin href="../../../static.files/${f}">`).join(""))</script><link rel="stylesheet" href="../../../static.files/normalize-9960930a.css"><link rel="stylesheet" href="../../../static.files/rustdoc-46132b98.css"><meta name="rustdoc-vars" data-root-path="../../../" data-static-root-path="../../../static.files/" data-current-crate="pyo3" data-themes="" data-resource-suffix="" data-rustdoc-version="1.85.1 (4eb161250 2025-03-15)" data-channel="1.85.1" data-search-js="search-75f5ac3e.js" data-settings-js="settings-0f613d39.js" ><script src="../../../static.files/storage-59e33391.js"></script><script defer src="../sidebar-items.js"></script><script defer src="../../../static.files/main-5f194d8c.js"></script><noscript><link rel="stylesheet" href="../../../static.files/noscript-893ab5e7.css"></noscript><link rel="alternate icon" type="image/png" href="../../../static.files/favicon-32x32-6580c154.png"><link rel="icon" type="image/svg+xml" href="../../../static.files/favicon-044be391.svg"></head><body class="rustdoc mod"><!--[if lte IE 11]><div class="warning">This old browser is unsupported and will most likely display funky things.</div><![endif]--><nav class="mobile-topbar"><button class="sidebar-menu-toggle" title="show sidebar"></button></nav><nav class="sidebar"><div class="sidebar-crate"><h2><a href="../../../pyo3/index.html">pyo3</a><span class="version">0.24.1</span></h2></div><div class="sidebar-elems"><section id="rustdoc-toc"><h2 class="location"><a href="#">Module asyncio</a></h2><h3><a href="#structs">Module Items</a></h3><ul class="block"><li><a href="#structs" title="Structs">Structs</a></li></ul></section><div id="rustdoc-modnav"><h2><a href="../index.html">In pyo3::<wbr>exceptions</a></h2></div></div></nav><div class="sidebar-resizer"></div><main><div class="width-limiter"><rustdoc-search></rustdoc-search><section id="main-content" class="content"><div class="main-heading"><span class="rustdoc-breadcrumbs"><a href="../../index.html">pyo3</a>::<wbr><a href="../index.html">exceptions</a></span><h1>Module <span>asyncio</span><button id="copy-path" title="Copy item path to clipboard">Copy item path</button></h1><rustdoc-toolbar></rustdoc-toolbar><span class="sub-heading"><a class="src" href="../../../src/pyo3/exceptions.rs.html#802">Source</a> </span></div><details class="toggle top-doc" open><summary class="hideme"><span>Expand description</span></summary><div class="docblock"><p>Exceptions defined in Python’s <a href="https://docs.python.org/3/library/asyncio.html"><code>asyncio</code></a>
module.</p>
</div></details><h2 id="structs" class="section-header">Structs<a href="#structs" class="anchor">§</a></h2><ul class="item-table"><li><div class="item-name"><a class="struct" href="struct.CancelledError.html" title="struct pyo3::exceptions::asyncio::CancelledError">Cancelled<wbr>Error</a></div><div class="desc docblock-short">A Rust type representing an exception defined in Python code.</div></li><li><div class="item-name"><a class="struct" href="struct.IncompleteReadError.html" title="struct pyo3::exceptions::asyncio::IncompleteReadError">Incomplete<wbr>Read<wbr>Error</a></div><div class="desc docblock-short">A Rust type representing an exception defined in Python code.</div></li><li><div class="item-name"><a class="struct" href="struct.InvalidStateError.html" title="struct pyo3::exceptions::asyncio::InvalidStateError">Invalid<wbr>State<wbr>Error</a></div><div class="desc docblock-short">A Rust type representing an exception defined in Python code.</div></li><li><div class="item-name"><a class="struct" href="struct.LimitOverrunError.html" title="struct pyo3::exceptions::asyncio::LimitOverrunError">Limit<wbr>Overrun<wbr>Error</a></div><div class="desc docblock-short">A Rust type representing an exception defined in Python code.</div></li><li><div class="item-name"><a class="struct" href="struct.QueueEmpty.html" title="struct pyo3::exceptions::asyncio::QueueEmpty">Queue<wbr>Empty</a></div><div class="desc docblock-short">A Rust type representing an exception defined in Python code.</div></li><li><div class="item-name"><a class="struct" href="struct.QueueFull.html" title="struct pyo3::exceptions::asyncio::QueueFull">Queue<wbr>Full</a></div><div class="desc docblock-short">A Rust type representing an exception defined in Python code.</div></li><li><div class="item-name"><a class="struct" href="struct.TimeoutError.html" title="struct pyo3::exceptions::asyncio::TimeoutError">Timeout<wbr>Error</a></div><div class="desc docblock-short">A Rust type representing an exception defined in Python code.</div></li></ul></section></div></main></body></html>